File: app/src/main/java/com/websarva/wings/android/kaigonote/NitizyoukirokuActivity.java
package com.websarva.wings.android.kaigonote;

import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

import com.websarva.wings.android.kaigonote.databinding.NitizyoukirokuBinding;

public class NitizyoukirokuActivity extends AppCompatActivity {
    private NitizyoukirokuBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = NitizyoukirokuBinding.inflate(getLayoutInflater());
        View view = binding.getRoot();
        setContentView(view);
    }
}

File: app/src/main/java/com/websarva/wings/android/kaigonote/NitizyoukirokuActivity.kt
package com.websarva.wings.android.kaigonote

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.websarva.wings.android.kaigonote.databinding.NitizyoukirokuBinding

class NitizyoukirokuActivity : AppCompatActivity() {
    private var binding: NitizyoukirokuBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = NitizyoukirokuBinding.inflate(layoutInflater)
        val view: View = binding!!.root
        setContentView(view)
    }
}